versioned_docs/version-2.0-2.4/getting-started/installation-and-upgrade/advanced-options/advanced-use-cases/air-gap-helm2/install-rancher.md

```diff
@@ -2,6 +2,10 @@
 title: 4. Install Rancher
 ---
 
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/install-rancher-ha"/>
```

Comment:
Shouldn't this be added to `versioned_docs/version-2.0-2.4/getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/install-rancher-ha` instead?

Reply:
I'll make the correction

Reply:
Fixed in [eeebdfd](https://github.com/rancher/rancher-docs/pull/623/commits/eeebdfd7d95c4b3f833d711e488769de8bf664b3)